### ISSUE TITLE
Incorrect counting of machines

### ISSUE THREAD

**Issue by GuiltyBystander** [Helfima/helmod#13]
When it calculates how many machines are needed, it doesn't account for productivity.

Quick way to see this is to add 2 Productivity modules (-15% speed each) and 1 Tier 2 Speed module (+30%) speed. This is a total net speed of +0%. It thinks this setup takes the same number of assemblers as having 0 modules.

**Comment by Helfima** [comment]
hello,
if your assembly machine produce 100 items/m
with 2 Productivity modules (-15% speed each) and 1 Tier 2 Speed module (+30%) speed it produce 108 items/m
you need less machines to produce 100 items/m

**Comment by GuiltyBystander** [comment]
You're right, it should need less machines, but helmod doesn't report that.
With no modules, need 16 and 24 assemblers.
![image](https://cloud.githubusercontent.com/assets/3122092/20143200/e3483e60-a64c-11e6-8304-f27220d3924f.png)

With productivity but 0% net speed.  Needs 16 and 20 assemblers.  I didn't test this when I first reported, I just noticed that this but only affects the top recipe.
![image](https://cloud.githubusercontent.com/assets/3122092/20143229/07c2c9ae-a64d-11e6-97dd-53fe297c4780.png)
